AndroidApps/BundleTransport/app/src/main/java/net/discdd/bundletransport/wifi/DDDWifiServer.java

```diff
@@ -263,6 +283,38 @@ void processDeviceInfo(WifiP2pDevice wifiP2pDevice) {
         }
     }
 
+    @SuppressLint("MissingPermission")
+    public void updateRecencyBlob(GetRecencyBlobResponse recencyBlob) {
+        var encodedBlob = Base64.getEncoder().encodeToString(recencyBlob.toByteArray());
+        if (serviceInfo == null || txt == null) {
+            return;
+        } else if (encodedBlob.length() > 255) {
+            logger.log(WARNING, "Recency blob too large to update in Wi-Fi Direct service TXT record, size: " +
+                    encodedBlob.length());
```

Comment:
i wonder if there is some sort of error blob that we can put in so that it can sort of work instead of failing silently.